mlc_llm/core.py

```diff
@@ -793,6 +793,8 @@ def build_model_from_args(args: argparse.Namespace):
     else:
         with open(os.path.join(args.model_path, "config.json"), encoding="utf-8") as i_f:
             config = json.load(i_f)
+    if 'vocab_size' not in config and 'padded_vocab_size' in config:
```

Comment:
Thanks @Jasonsey, could you please remove these 2 lines. This seems to be implemented below:

https://github.com/mlc-ai/mlc-llm/blob/main/mlc_llm/core.py#L885-L891

Reply:
After testing, this problem has indeed been fixed in the latest code and has been submitted for deletion.